drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -38,6 +40,7 @@ config ETH_STM32_HAL_USE_DTCM_FOR_DMA_BUFFER
 	bool "Use DTCM for DMA buffers"
 	default y
 	depends on SOC_SERIES_STM32F7X
+	depends on !SOC_SERIES_STM32H7X
```

Comment:
Due to F7 dependency, it is useless to introduce !H7
```suggestion
```

Reply:
The line were removed.